ui/tests/acceptance/tools-test.js

```diff
@@ -101,7 +101,7 @@ test('tools functionality', function(assert) {
   //random
   click('[data-test-tools-action-link="random"]');
   andThen(() => {
-    assert.equal(find('[data-test-tools-input="bytes"]').val(), 32, 'defaults to 32 bytes');
+    assert.dom('[data-test-tools-input="bytes"]').hasValue(32, 'defaults to 32 bytes');
```

Comment:
This needs to be `hasValue('32'` because QUnit's `assert.equal` uses `==`, it was passing previously 😬